### ISSUE TITLE
aws-cli/setup fails on alpine

### ISSUE THREAD

**Issue by Almenon** [CircleCI-Public/aws-cli-orb#143]
```yml
version: 2.1
executors:
  terraform:
    docker:
      - image: hashicorp/terraform:1.3.4
    environment:
      AWS_DEFAULT_REGION: us-west-1
orbs:
  aws-cli: circleci/aws-cli@4.0.0
commands:
  setup:
    steps:
      - checkout
      - aws-cli/setup:
          role_arn: 'YOUR-ROLE-ARN-HERE'


jobs:
  Plan-terraform:
    executor: terraform
    steps:
      - setup

workflows:
  Plan PR:
    jobs:
      - Plan-terraform:
          context: DevOps # no-op empty context
          filters:
            branches:
              ignore:
                - main
```

ERROR:
```
Installing AWS CLI v2
fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/community/x86_64/APKINDEX.tar.gz
(1/4) Installing libgcc (11.2.1_git20220219-r2)
  0%                                             (2/4) Installing libstdc++ (11.2.1_git20220219-r2)
  0%                                             (3/4) Installing binutils (2.38-r3)
 16% #######                                      17% #######                                      22% ##########                                   23% ##########                                   35% ###############                              36% ###############                              36% ################                             48% #####################                        49% #####################                        50% ######################                       53% #######################                      58% #########################                    59% ##########################                   60% ##########################                   70% ###############################              71% ###############################              73% ################################             83% ####################################         84% ####################################         84% #####################################        85% #####################################        86% #####################################        86% ######################################       87% ######################################       88% ######################################       88% #######################################      89% #######################################      90% #######################################      90% ########################################     91% ########################################     92% ########################################     93% ########################################     93% #########################################    94% #########################################    95% #########################################    95% ##########################################   96% ##########################################   97% ##########################################  (4/4) Installing curl (8.1.2-r0)
 97% ########################################### % ############################################Executing busybox-1.35.0-r17.trigger
OK: 38 MiB in 35 packages
curl: (48) An unknown option was passed in to libcurl

Exited with code exit status 48
```

Adding `      - run: apk --no-cache add libcurl;apk --no-cache upgrade libcurl;` before aws-cli/setup fixes it.

**Comment by brivu** [comment]
Hey @Almenon - I am trying this on `Alpine` and it seems to work. Let test this with the `Terraform` container and get back to you. 

Thank you again!

**Comment by brivu** [comment]
Hey @Almenon - I got a fix for this in place in PR #144 . Thanks for letting me know how you fixed it!

**Comment by Almenon** [comment]
welcome :)
